lib/facility_factory.rb

```diff
@@ -0,0 +1,52 @@
+require "spec_helper"
```

Comment:
Typically our spec helper to should just that - a helper for specs. I would manually require the lib files you need here so you aren't requiring extra things you don't need unnecessarily. Your spec helper is small now. It's gonna get real big real fast once you move into mod 2 with a whole bunch of other setup stuff for testing tools that you'll be using. We don't need or want to waste computing resources loading all that when we aren't actually testing our program.